Update AlphaAnimals_CE_Patch_Projectiles.xml

Breadbox2k19 opened the pull request:
Animal range attack are much like bows and arrow than bullets, hence I nerfed most of them in terms of speed, penetration but sometimes, higher damage. Increased gallatross damage and increased fragmentation, since a reduction from radii 2 to 1 result in a huge reduction in blast radius(9 tiles to 1 tile).

Feel free to discuss.

Breadbox2k19 committed:
Update AlphaAnimals_CE_Patch_Projectiles.xml

Animal range attack are much like bows and arrow than bullets, hence I nerfed most of them in terms of speed, penetration but sometimes, higher damage. Increased gallatross damage and increased fragmentation, since a reduction from radii 2 to 1 result in a huge reduction in blast radius(9 tiles to 1 tile).

Feel free to discuss.

Juanfrank committed:
Update AlphaAnimals_CE_Patch_Projectiles.xml

Juanfrank commented:
Made some changes to speed, damages to projectile equivalents. Reduced Gallatross' rock sharp penetration since an ordinary rock wouldn't have piercing attributes, and added some blunt penetration. Also speed to make it have more of a "floaty" appareance, since it's launched by telekinesis.

This warrants some testing since fragments can be hella deadly especially big fragments.

Breadbox2k19 commented:
I would reduce the speed of most animal projectiles, they easily 2-3 times the speed of arrows, especially nightling which fires a very large javelin like spike, 20 speed is appropriate and doesn’t need to be faster.

Juanfrank commented:
That's actually true

Juanfrank committed:
Update AlphaAnimals_CE_Patch_Projectiles.xml

Breadbox2k19 committed:
Update AlphaAnimals_CE_Patch_Projectiles.xml

Barbslinger barb is meant to be noticeably faster than others.
Quill goes back to 20 speed.
12 speed is way too slow, increased a little to make less easy to avoid.

Juanfrank committed:
Update AlphaAnimals_CE_Patch_Projectiles.xml
